templates/hooks/cluster/sdk_file_end.go.tpl

```diff
@@ -0,0 +1,26 @@
+{{ $CRD := .CRD }}
+{{ $SDKAPI := .SDKAPI }}
+
+{{- $updateClusterVersion := (index $SDKAPI.API.Operations "UpdateClusterConfig") -}}
```

Comment:
what does "Version" refer to in the above `$updateClusterVersion` variable name?